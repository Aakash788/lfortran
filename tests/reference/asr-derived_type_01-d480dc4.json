{
    "basename": "asr-derived_type_01-d480dc4",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/derived_type_01.f90",
    "infile_hash": "74c9f7ee5dfaf581b24988b384a68b11d6023d84b46344260436f624",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-derived_type_01-d480dc4.stderr",
    "stderr_hash": "42443d00dba125e637373afac03495fcb7454ea3a4b824acd8e6fb4c",
    "returncode": 2
}