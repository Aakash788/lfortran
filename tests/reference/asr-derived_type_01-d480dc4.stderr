semantic error: Interface for deferred binding not found
 --> tests/errors/derived_type_01.f90:7:9
  |
7 |         procedure(testing_interface), deferred  :: testing
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
